common/app/views/fragments/image.scala.html

```diff
@@ -23,6 +36,13 @@
     @* IE 9 needs this workaround as per https://scottjehl.github.io/picturefill/ *@
     <!--[if IE 9]><video style="display: none;"><![endif]-->
 
+    @if(isImmersiveMainMedia) {
+        @picture.largestImage.map { largestImage =>
+          @immersivePortraitSource(largestImage)
+            @immersivePortraitSource(largestImage, hidpi = true)
```

Comment:
@NataliaLKB I think if you switch the order of these two it will fix the issue @paperboyo mentioned.